src/main/java/com/karankumar/bookproject/backend/service/PublisherService.java

```diff
@@ -0,0 +1,59 @@
+package com.karankumar.bookproject.backend.service;
```

Comment:
Could you please add our [copyright notice](https://github.com/Project-Books/book-project/blob/master/docs/COPYRIGHT) to the start of this file?